prodtests/full-system-test/workflow-setup.sh

```diff
@@ -1,140 +1,8 @@
 #!/bin/bash
 
-# used to avoid sourcing this file 2x
-if [[ -z $SOURCE_GUARD_WORKFLOW_SETUP ]]; then
-SOURCE_GUARD_WORKFLOW_SETUP=1
-
-# ---------------------------------------------------------------------------------------------------------------------
-#Some additional settings used in this workflow
-
-if [[ $SYNCMODE == 1 ]]; then
-  if [[ -z "${WORKFLOW_DETECTORS_MATCHING+x}" ]]; then export WORKFLOW_DETECTORS_MATCHING="ITSTPC,ITSTPCTRD,ITSTPCTOF,ITSTPCTRDTOF,PRIMVTX"; fi # Select matchings that are enabled in sync mode
-else
-  if [[ -z "${WORKFLOW_DETECTORS_MATCHING+x}" ]]; then export WORKFLOW_DETECTORS_MATCHING="ALL"; fi # All matching / vertexing enabled in async mode
-fi
-
-MID_FEEID_MAP="$FILEWORKDIR/mid-feeId_mapper.txt"
-
-ITSMFT_STROBES=""
-[[ ! -z $ITS_STROBE ]] && ITSMFT_STROBES+="ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;"
-[[ ! -z $MFT_STROBE ]] && ITSMFT_STROBES+="MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
-
-LIST_OF_ASYNC_RECO_STEPS="MID MCH MFT FDD FV0 ZDC HMP"
-
-DISABLE_DIGIT_ROOT_INPUT="--disable-root-input"
-if [[ -z ${DISABLE_DIGIT_CLUSTER_INPUT+x} ]]; then DISABLE_DIGIT_CLUSTER_INPUT="--clusters-from-upstream"; fi
-
-# ---------------------------------------------------------------------------------------------------------------------
-# Set active reconstruction steps (defaults added according to SYNCMODE)
-
-has_processing_step()
-{
-  [[ $WORKFLOW_EXTRA_PROCESSING_STEPS =~ (^|,)"$1"(,|$) ]]
-}
-
-for i in `echo $LIST_OF_GLORECO | sed "s/,/ /g"`; do
-  has_processing_step MATCH_$i && add_comma_separated WORKFLOW_DETECTORS_MATCHING $i # Enable extra matchings requested via WORKFLOW_EXTRA_PROCESSING_STEPS
-done
-if [[ $SYNCMODE == 1 ]]; then # Add default steps for synchronous mode
-  add_comma_separated WORKFLOW_EXTRA_PROCESSING_STEPS ENTROPY_ENCODER
-else # Add default steps for async mode
-  for i in $LIST_OF_ASYNC_RECO_STEPS; do
-    has_detector_reco $i && add_comma_separated WORKFLOW_EXTRA_PROCESSING_STEPS ${i}_RECO
-  done
-  add_comma_separated WORKFLOW_EXTRA_PROCESSING_STEPS TPC_DEDX
+if [ "0$O2DPG_ROOT" == "0" ]; then
+    echo O2DPG environment not loaded
```

Comment:
please add `1>&2`

Reply:
Done and also in the other scripts where it was missing where I took that snippet from